common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -209,38 +219,43 @@ function () {
                     // will give focus to Play/Pause button and tabbing
                     // forward to Volume button.
                     break;
-                // Close menu, give focus to button and change
-                // file type.
+
                 case KEY.ENTER:
                 case KEY.SPACE:
                     this.button.focus();
                     this.changeFileType.call(this, event);
                     _closeMenu(this);
                     break;
-                // Close menu and give focus to speed control.
+
                 case KEY.ESCAPE:
                     _closeMenu(this);
                     this.button.focus();
                     break;
             }
             return false;
-        }
-        else {
+
+        } else if(target.is('.has-dropdown')) {
+
+            switch(keyCode) {
+                case KEY.DOWN:
+                    _focusOnFirst(this);
+                    break;
+            }
+            return false;
+        } else {
+
             switch(keyCode) {
-                // Open menu and focus on last element of list above it.
                 case KEY.ENTER:
                 case KEY.SPACE:
                 case KEY.UP:
                     _openMenu(this);
                     this.menuItemsLinks.last().focus();
                     break;
-                // Close menu.
+
                 case KEY.ESCAPE:
                     _closeMenu(this);
                     break;
             }
-            // We do not stop propagation and default behavior on a TAB
```

Comment:
They were interrupting my thought process. I've replaced since I'm done in here.